coralproject/talk
# [CORL-1524] Add submit comment shortcut: Ctrl+Enter

## What does this PR do?

Make it so you can submit a comment by while typing by hitting `Ctrl + Enter`.

**Note: this functionality is dependent on other changes in the RTE repo -> https://github.com/coralproject/rte/pull/12**

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Build and pull in changes from https://github.com/coralproject/rte/pull/12
- Type a comment
- Press `Ctrl + Enter` instead of hitting Submit
- New comment should get posted
